pkg/kv/kv_object_format.go

```diff
@@ -0,0 +1,54 @@
+package kv
+
+import (
+	"google.golang.org/protobuf/encoding/protojson"
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/reflect/protoreflect"
+)
+
+var matchers = make(map[string]protoreflect.ProtoMessage)
```

Comment:
Can we use a simple list of prefixes (just for a start) and not regex?

Reply:
That would prevent us from identifying everything that is more complex than pref+ID, such as secondary indexes (user-groups, group-policies, user-credentials)
I think it will also require specific handling for const paths (installaion ID, installation timestamp)
We need to decide the risk of using these regexes, over the benefits they bring. I'm specifically aiming at future types we will need to introduce. WDYT?

Reply:
Done - removed all regexes as the type can be identified by the prefix

Reply:
Update - regexes are back due to current inability to define actions prefix